src/test/java/org/dependencytrack/persistence/converter/ToolsJsonConverterTest.java

```diff
@@ -198,7 +206,11 @@ public void testConvertToAttribute() {
                 {
                   "components": [
                     {
-                       "author": "componentAuthor",
```

Comment:
For the sake of backward-compatibility, we should leave this `author` field here. Since existing records in the database will have `author` populated, but not `authors`. We should verify that that won't blow up.